stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/util/extensions/MessageOptionItemVisibility.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2014-2025 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.compose.util.extensions
+
+import io.getstream.chat.android.compose.ui.components.messageoptions.MessageOptionItemVisibility
+import io.getstream.chat.android.models.Message
+import io.getstream.chat.android.models.User
+
+internal fun MessageOptionItemVisibility.canReplyToMessage(
+    message: Message,
+    ownCapabilities: Set<String>,
+): Boolean = io.getstream.chat.android.ui.common.utils.canReplyToMessage(
```

Comment:
Can we import these functions?

Reply:
They can be imported because they have different parameters:

<img width="561" alt="Screenshot 2025-02-19 at 08 22 38" src="https://github.com/user-attachments/assets/c1455661-6a98-40b9-b9d8-c86811840b5c" />

Reply:
I am not sure why I couldn't while refactoring this code :thinking: ....
NVM, here are the changes ;) https://github.com/GetStream/stream-chat-android/pull/5640